opennms-doc/guide-install/src/asciidoc/text/minion/advanced-configuration.adoc

```diff
@@ -0,0 +1,81 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+=== Advanced Minion Configuration
+
+This section provides information for advanced configuration topics, such as _Syslog_-, _SNMP Trap_ receiving or running in non-root environments.
+
+[[gi-install-minion-rhel-ping]]
+==== Configure Linux to Allow Non-Root ICMP
+
+By default, _Linux_ does not allow regular users to perform `ping` operations from arbitrary programs (including Java).
+To enable the _Minion_ to ping properly, you must set a `sysctl` option.
+
+.Enable User Ping (Running System)
+[source, shell]
+----
+# run this command as root to allow ping by any user (does not survive reboots)
+sysctl net.ipv4.ping_group_range='0 429496729'
+----
+
+If you wish to restrict the range further, use the _GID_ for the user the _Minion_ will run as, rather than `429496729`.
+
+To enable this permanently, create a file in `/etc/sysctl.d/` to set the range:
+
+./etc/sysctl.d/99-zzz-non-root-icmp.conf
+[source, shell]
+----
+# we start this filename with "99-zzz-" to make sure it's last, after anything else that might have set it
+net.ipv4.ping_group_range=0 429496729
+----
+
+[[gi-install-minion-rhel-trapd]]
+==== Configure Minion to Receive Traps
+
+If you wish your Minion to listen to SNMP traps, you will need to configure your firewall to port forward from the privileged trap port (162) to the Minion's default trap listener on port 1162.
+
+.Forward 1162 to 162 with Firewalld
+[source, shell]
+----
+# enable masquerade to allow port-forwards
+firewall-cmd --add-masquerade
+# forward port 162 TCP and UDP to port 162 on localhost
+firewall-cmd --add-forward-port=port=162:proto=udp:toport=1162:toaddr=127.0.0.1
+firewall-cmd --add-forward-port=port=162:proto=tcp:toport=1162:toaddr=127.0.0.1
+----
+
+[[gi-install-minion-rhel-syslogd]]
+==== Configure Minion to Receive Syslog Messages
+
+If you wish your Minion to listen to syslog messages, you will need to configure your firewall to port forward from the privileged syslog port (514) to the Minion's default syslog listener on port 1514.
+
+.Forward 1514 to 514 with Firewalld
+[source, shell]
+----
+# enable masquerade to allow port-forwards
+firewall-cmd --add-masquerade
+# forward port 514 TCP and UDP to port 514 on localhost
+firewall-cmd --add-forward-port=port=514:proto=udp:toport=1514:toaddr=127.0.0.1
+firewall-cmd --add-forward-port=port=514:proto=tcp:toport=1514:toaddr=127.0.0.1
+----
+
+[[gi-install-minion-config-file]]
+==== Minion Configuration File
+
+Allowing to deploy _Minions_ through configuration management tools a configuration file instead of the _Karaf CLI_ can be used.
```

Comment:
Beside manually configuring a _Minion_ instance via the _Karaf CLI_ you can also modify and deploy its configuration file through configuration management tools.